pages.zh/linux/lvs.md

```diff
@@ -0,0 +1,33 @@
+# lvs
+
+> 显示逻辑卷信息.
+> 另见: `lvm`.
+> 更多信息: <https://man7.org/linux/man-pages/man8/lvs.8.html>.
+
+- 显示逻辑卷信息:
```

Comment:
```suggestion
- 显示逻辑卷信息：
```

Reply:
@NealSCarffery Looks like you applied the changes, but you reverted them with the commit `567ba28caaff755114add495480872247fec725a`.

You can apply suggestions from this view without the need to make the commits manually.
Take a look here: <https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes>

If you need help ask us!

Reply:
```suggestion
- 显示逻辑卷信息：
```